api/src/gcnotfiy-services/gcnotify-service.ts

```diff
@@ -0,0 +1,97 @@
+import axios from 'axios';
+import { ApiError, ApiErrorType } from '../errors/custom-error';
+import { IgcNotfiyPostReturn, IgcNotfiyGenericMessage } from '../models/gcnotify';
+
```

Comment:
Can you change the folder this is in from `gcnofity-services` to just `services`.

Reply:
The name of this file is correct though.